libfreerdp/core/nla.c

```diff
@@ -159,9 +159,14 @@ static void nla_identity_free(SEC_WINNT_AUTH_IDENTITY* identity);
 
 #define ber_sizeof_sequence_octet_string(length) \
 	ber_sizeof_contextual_tag(ber_sizeof_octet_string(length)) + ber_sizeof_octet_string(length)
-#define ber_write_sequence_octet_string(stream, context, value, length)                \
-	ber_write_contextual_tag(stream, context, ber_sizeof_octet_string(length), TRUE) + \
-	    ber_write_octet_string(stream, value, length)
+
+static inline size_t ber_write_sequence_octet_string(wStream* stream, BYTE context, void* value,
```

Comment:
Please use the `INLINE` macro to support older compilers

Reply:
`ber_write_octet_string` uses `const BYTE*` for `value`, please also use this here.

Reply:
Thanks for the review. Changes made.